setup.cfg

```diff
@@ -17,4 +17,5 @@ pep8ignore =
     E241    # multiple spaces after ','
     E226    # missing white space around arithmetic operator
     E222    # multiple spaces after operator
+    W391    # blank line at the end of file
```

Comment:
Why ignoring ?

Reply:
tentacles package manager is not creating eof blank lines in init files, it makes pytest fail on this lint check

Reply:
ok :smile: